docker-compose.yml

```diff
@@ -113,26 +75,71 @@ services:
       - AWS_ACCESS_KEY_ID=.
       - AWS_SECRET_ACCESS_KEY=.
     depends_on:
-      - ipfs
+      - cas_ipfs
       - database
       - localstack
       - cas_api
       - launch_contract
       - ganache
       - cas
-    volumes:
-      - './migrations:/cas/migrations'
     networks:
       - internal
     deploy:
       mode: replicated
-      replicas: 2
+      replicas: 1
 
-  ipfs:
+  cas_api:
+    image: ceramicnetwork/ceramic-anchor-service:release-candidate
+    container_name: cas_api
+    environment:
+      - NODE_ENV=dev
+      - APP_MODE=server
+      - APP_PORT=8081
+      - ANCHOR_CONTROLLER_ENABLED=false
+      - ANCHOR_EXPIRATION_PERIOD=0
+      - ANCHOR_SCHEDULE_EXPRESSION=0/1 * * * ? *
+      - BLOCKCHAIN_CONNECTOR=ethereum
+      - ETH_GAS_LIMIT=4712388
+      - ETH_GAS_PRICE=100000000000
+      - ETH_NETWORK=ganache
+      - ETH_RPC_URL=http://ganache:8545
+      - ETH_OVERRIDE_GAS_CONFIG=false
+      - ETH_WALLET_PK=0x06dd0990d19001c57eeea6d32e8fdeee40d3945962caf18c18c3930baa5a6ec9
+      - ETH_CONTRACT_ADDRESS=0xd3f84cf6be3dd0eb16dc89c972f7a27b441a39f2
+      - IPFS_API_URL=http://cas_ipfs:5001
+      - IPFS_API_TIMEOUT=120000
+      - LOG_LEVEL=debug
+      - LOG_TO_FILES=true
+      - LOG_PATH=/usr/local/var/log
+      - MERKLE_DEPTH_LIMIT=0
+      - DB_NAME=anchor_db
+      - DB_HOST=database
+      - DB_PORT=5432
+      - DB_USERNAME=root
+      - DB_PASSWORD=root
+      - VALIDATE_RECORDS=false
+    depends_on:
+      cas_ipfs:
+        condition: service_healthy
+      database:
+        condition: service_healthy
+    ports:
+      - '8081:8081'
+    networks:
+      - internal
+    healthcheck:
+      test: ['CMD', 'curl', 'http://localhost:8081/api/v0/healthcheck']
```

Comment:
ohohohoh i see what you mean yes. But yes the server doesn't start until the db connection is established and migrations will run =) But i can replace the migration volume above just in case something changes

Reply:
If the server doesn't start till the migrations are done, then you shouldn't need to add that back :) In fact, would you mind also removing the migrations folder from the repo? It should no longer be necessary either.

Reply:
done =)